ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeers.java

```diff
@@ -203,7 +203,9 @@ public Stream<EthPeer> streamAllPeers() {
   }
 
   public Stream<EthPeer> streamAvailablePeers() {
-    return streamAllPeers().filter(EthPeer::readyForRequests);
+    return streamAllPeers()
+        .filter(EthPeer::readyForRequests)
+        .filter(peer -> !peer.isDisconnected());
```

Comment:
seems sensible enough.  This would be additional protection from a race between besu disconnecting a peer and subsequently responding to that disconnect?

Reply:
No sure I got your question, but this part is not affecting the way we process incoming messages